### PR TITLE
fix(suite-native): account import error handling

### LINKED ISSUE #8557: Not able to import some LTC account types to Lite app
**Describe the bug**
Some account types of Litecoin are not imported.
Overall import looks as expected but they are not listed in `My assets` section
Tested on all seed standard accounts.
First two accounts are not imported and third one is imported

**Info:**

-   Suite Version: lite app Tested in 23.5.1 (425) on Android os
-   OS:Android

**How to reproduce**

1. Try to import provided XPUBs
2. Observe imported accounts after you finish import

**Screenshots**
image
image
image

### PR DESCRIPTION
**Pull request by PeKne**
## Description

- Error message is displayed if error occurs during the import.

## Related Issue

Resolve #8557 

## Screenshots
Screenshot 2023-05-29 at 14 10 25

### CODE REVIEW

File: suite-native/module-accounts-import/src/useShowImportError.ts [hunk 1]
```diff
@@ -25,46 +20,51 @@ const alertErrorMap: Record<AlertError, AlertErrorOptions> = {
...
 const pictogramVariant: PictogramVariant = 'red';
 
 export const useShowImportError = (
     networkSymbol: NetworkSymbol,
-    navigation: StackToTabCompositeNavigationProp<
-        AccountsImportStackParamList,
-        AccountsImportStackRoutes.AccountImportLoading,
-        RootStackParamList
-    >,
+    navigation: NavigationProp<any>,
```
Comment (juriczech) [inline suite-native/module-accounts-import/src/useShowImportError.ts]: missing navigation prop

**Review by juriczech — CHANGES_REQUESTED** [review]

File: suite-native/module-accounts-import/src/useShowImportError.ts (hunk 1 above)
Comment (PeKne) [reply suite-native/module-accounts-import/src/useShowImportError.ts]: It is intentional. So you can send any `NavigationProp` in there.

Comment (juriczech) [reply suite-native/module-accounts-import/src/useShowImportError.ts]: You shouldn't use any navigation prop. It's import error, so the origin will always be `AccountsImportStack` and you can then type the paths we want to allow for this hook to redirect to. You should almost never need to use `any` for this kind of thing. 

Either make it more flexible by some nice TS generic type, or type it properly but remove `any` please.

Comment (PeKne) [reply suite-native/module-accounts-import/src/useShowImportError.ts]: fixed in c5936dbbb

**Review by Nodonisko — APPROVED** [review]

**Review by juriczech — APPROVED** [review]

**Pull request merged**